### DISCUSSION TITLE
Vulnerability Issue

### DISCUSSION THREAD

**Discussion by Tring-Dharshan** [MichalLytek/type-graphql#1470] [Q&A]
vulnerability reported in inflight(last version was release in Oct 2016) package used by type-graphql, kindly let me know if there is a way to resolve this.

<img width="158" alt="Screenshot 2023-07-12 162436" src="https://github.com/MichalLytek/type-graphql/assets/102800724/c44ab5de-cdf3-4aee-be27-74f22da53a73">

I'm facing vulnerability issue in the type-graphql dependency inflight . Is there any solution to resolve this vulnerability issue?

**Comment by carlocorradini** [comment]
Can you add more information please. Thanks

**Reply by AChangXD** [reply]
Hmm inflight was last published in 2016??
